packages/next-intl/.size-limit.ts

```diff
@@ -2,43 +2,79 @@ import type {SizeLimitConfig} from 'size-limit';
 
 const config: SizeLimitConfig = [
   {
+    name: 'import * from \'next-intl\' (react-client)',
     path: 'dist/production/index.react-client.js',
     limit: '14.095 KB'
   },
   {
+    name: 'import * from \'next-intl\' (react-server)',
     path: 'dist/production/index.react-server.js',
     limit: '14.665 KB'
   },
   {
+    name: 'import {createSharedPathnamesNavigation} from \'next-intl/navigation\' (react-client)',
     path: 'dist/production/navigation.react-client.js',
-    limit: '3.155 KB'
+    import: '{createSharedPathnamesNavigation}',
+    limit: '3.825 KB'
   },
   {
+    name: 'import {createLocalizedPathnamesNavigation} from \'next-intl/navigation\' (react-client)',
+    path: 'dist/production/navigation.react-client.js',
+    import: '{createLocalizedPathnamesNavigation}',
+    limit: '3.825 KB'
+  },
+  {
+    name: 'import {createNavigation} from \'next-intl/navigation\' (react-client)',
+    path: 'dist/production/navigation.react-client.js',
+    import: '{createNavigation}',
+    limit: '3.825 KB'
```

Comment:
I think tree shaking in size limit has a bug. If only `createNavigation` is exported, the size amounts to 2.95 kB.

For comparison, exporting only `createSharedPathnamesNavigation` accounts for 2.97 kB and `createLocalizedPathnamesNavigation` for 3.24 kB.